db/migrate/20240823190536_reset_all_project_cache_counters.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# Migration to reset samples_count in all projects
```

Comment:
Can you update the comment here to clarify that resetting the samples_count here means that the actual count of project samples will be set for each project's samples_count